backend/controllers/searchExternalUsers.js

```diff
@@ -14,7 +17,7 @@ const searchFactory = (
   searchTitle,
   allowDownload,
 ) => {
-  const index = async (req, res) => {
+  return async (req, res) => {
```

Comment:
we no longer have separate index and results as we always do a search for users (this matches dps user search)